etc/skeleton/driver/normalizer/annotation.go

```diff
@@ -19,15 +18,6 @@ var Native = Transformers([][]Transformer{
 	},
 }...)
 
-// Code is a special block of transformations that are applied at the end
-// and can access original source code file. It can be used to improve or
-// fix positional information.
-//
-// https://godoc.org/gopkg.in/bblfsh/sdk.v2/uast/transformer/positioner
-var Code = []CodeTransformer{
```

Comment:
If this means that on a update of the driver this needs to be removed - let's add it to the SDK release notes, so a driver maintainer is not confused. WDYT?

Reply:
Sure, will mention it. Drivers should already use a new version of this transform stage, so it's only a matter of removing it in the driver as well.